packages/dashboard/e2e/tests/utils/grid.ts

```diff
@@ -0,0 +1,134 @@
+import { Locator, Page } from '@playwright/test';
+import last from 'lodash/last';
+
+import { DragPosition, dragAndDrop } from './dragAndDrop';
+import { center } from './mousePosition';
+import { BoundingBox, getBoundingBox } from './locator';
+
+export const GRID_SIZE = 10;
+
+const WidgetSelector = '[data-gesture=widget]';
+const SelectionSelector = '[data-gesture=selection]';
+const GridSelector = '#container';
+
+type ComponentLibrarySelector = 'kpi' | 'text';
+
+const WidgetSelectorMap: { [key in ComponentLibrarySelector]: string } = {
+  kpi: 'add KPI widget',
+  text: 'add Text widget',
+};
+
+type SelectionAnchor = 'bottom-right';
+
+const SelectionAnchorMap: { [key in SelectionAnchor]: string } = {
+  'bottom-right': '[data-anchor=bottom-right]',
+};
+
+export const gridUtil = (page: Page) => {
+  const dragGenerator = dragAndDrop(page);
+
+  const gridArea = page.locator(GridSelector);
+
+  return {
+    /**
+     *
+     * @param col column of grid to select
```

Comment:
Consider adding a hyphen to improve readability of @param doc comments, as recommended by [JSDoc](https://jsdoc.app/tags-param.html) and [TSDoc](https://tsdoc.org/pages/tags/param/). Comment applies to all usage of the tag.

Reply:
Yup can add those!